pkg/users/account-details.js

```diff
@@ -101,14 +101,10 @@ export function AccountDetails({ accounts, groups, current_user, user, shells })
     const [lastlogpath, setLastlogPath] = useState(null);
 
     useEffect(() => {
-        cockpit.spawn(["test", "-e", "/var/run/utmp"], { err: "ignore" }).then(() => {
-            setLastlogPath("/var/run/utmp");
+        getLastlogPath().then((path) => {
+            setLastlogPath(path);
         }).catch(() => {
```

Comment:
I think we don't need to catch errors here since getLastLogPath is not expected to throw an exception. And this could also be `async`... :-) `useEffect(() => setLastLogPath(await getLastLogPath()), [])` wdyt?